app/client/components/ParseOptions.ts

```diff
@@ -58,19 +62,33 @@ export function buildParseOptionsForm(
     return fromPairs(items.map((item) => [item.name, optionsMap.get(item.name)!.get()]));
   }
 
+  const labelsByName: {[key: string]: string} = {
+    lineterminator: t('Line terminator'),
+    include_col_names_as_headers: t('First row contains headers'),
+    delimiter: t('Field separator'),
+    skipinitialspace: t('Skip leading whitespace'),
+    quotechar: t('Quote character'),
+    doublequote: t('Quotes in fields are doubled'),
+    quoting: t('Convert quoted fields'),
+    escapechar: t('Escape character'),
+    start_with_row: t('Start with row'),
+    NUM_ROWS: t('Number of rows'),
+    encoding: t('Character encoding. See [the supported codecs]({{link}})', {link: 'https://tinyurl.com/py3codecs'})
```

Comment:
Note that I am not sure about how the SCHEMA's label are used in the [import_csv.py](https://github.com/gristlabs/grist-core/blob/f8c05bd161327fc3fd4c97299cca3cf9fc2c0511/sandbox/grist/imports/import_csv.py#L17) module.

I haven't dared to remove the `label` properties from there, I don't know whether I should.

Reply:
I see what you're saying. I would guess that the labels in SCHEMA are there for the purposes of the front end to read them. And I'd guess that removing them would be good, to avoid confusion. But I don't know that. Could you try and see if anything breaks?

Reply:
Thanks for sharing your thoughts on this!

Done here: c5165bb2e909a7ac818cf9448f81ac61070e093a

Reply:
I take a look at the CI errors, it sounds related to this commit